src/main/java/dev/slimevr/vr/processor/skeleton/LegTweakBuffer.java

```diff
@@ -0,0 +1,521 @@
+package dev.slimevr.vr.processor.skeleton;
+
+import com.jme3.math.Quaternion;
+import com.jme3.math.Vector3f;
+
+
+// class that holds data related to the state and other variuse attributes of the legs
+// such as the position of the foot, knee, and waist, after and before correction,
+// the velocity of the foot and the computed state of the feet at that frame.
+// mainly calculates the state of the legs per frame using these rules:
+// The conditions for an unlock are as follows:
+// 1. the foot is to far from its correct position
+// 2. a velocity higher than a threashold is achived
+// 3. a large acceleration is applied to the foot
+// 4. angular velocity of the foot goes higher than a threashold
+// The conditions for a lock are the opposite of the above
+// but require a lower value for all of the above conditions
```

Comment:
Shouldn't this be formatted like docs if you're explaining what the class does?
```
/**
 * documentation 
 */

Reply:
Yes it should be